### PR TITLE
make SoundDeviceSamplesGenerator always importable

### LINKED ISSUE #211: Improve handling of optional classes
There exist some optional imports in the [init.py](acoular/__init__.py#L19-27), which are problematic for the following reason:

* bare except is bad code style and a linting error (E722)
* classes are simply missing if the required packages are not installed (I think we should at least raise an ImportError when the class is imported by a user)

### PR DESCRIPTION
**Pull request by adku1173**
closes #211

### CODE REVIEW

File: acoular/__init__.py [hunk 1]
```diff
@@ -126,3 +122,4 @@
     WriteWAV,
 )
 from .trajectory import Trajectory
+from .version import __author__, __date__, __version__
```
Comment (artpelling) [inline acoular/__init__.py:125]: why is this needed?

**Review by artpelling — APPROVED** [review]

File: acoular/__init__.py (hunk 1 above)
Comment (adku1173) [reply acoular/__init__.py:125]: It is needed to call `acoular.__version__` etc.
